vboard-front/run_http_server.sh

```diff
@@ -7,10 +7,24 @@ if ! [ -d /var/www/vboard/compile ]; then
     exit 1
 fi
 
-echo 'Inserting $VBOARD_WP_PUBLIC_HOST or /vblog in config.js'
-sed -i "s~\$VBOARD_WP_PUBLIC_HOST~${VBOARD_WP_PUBLIC_HOST:-/vblog}~" /var/www/vboard/compile/scripts/config.js
-echo 'Inserting $VBOARD_LOCALISATIONS in config.js'
-sed -i "s/\$VBOARD_WP_PUBLIC_HOST/${VBOARD_LOCALISATIONS:-}/" /var/www/vboard/compile/scripts/config.js
+if [ -n "${VBOARD_API_ENDPOINT:-}" ]; then
+    echo 'Inserting $VBOARD_API_ENDPOINT in index.html'
+    sed -i "s/\$VBOARD_API_ENDPOINT/${VBOARD_API_ENDPOINT:-}/" /var/www/vboard/index.html
```

Comment:
VBOARD_API_ENDPOINT includes forward slashes, sed calls should use tildes like below

Reply:
Well spotted !